astropy/io/fits/tests/test_image.py

```diff
@@ -1010,7 +1010,9 @@ def test_comp_image_hcompress_image_stack(self):
         hdu.writeto(self.temp('test.fits'))
 
         with fits.open(self.temp('test.fits')) as hdul:
-            assert (hdul['SCI'].data == cube).all()
+            # HCOMPRESSed images are allowed to deviate from the original by
+            # about 1/quantize_level of the RMS in each tile. 
+            assert np.abs(hdul['SCI'].data - cube).max() < 1./15.
```

Comment:
1/16 was a bit too low, so that I assume there is an off-by-one somewhere in the docs or in my understanding.